core/controlplane/config/templates/cloud-config-controller

```diff
@@ -778,10 +778,23 @@ write_files:
           -ec \
           '
             cfn-signal -e 0 --region {{.Region}} --resource {{.Controller.LogicalName}} --stack "${{.StackNameEnvVarName}}"
-          '
-
+          ')
+      result_code=$?
+      echo $result
       rkt rm --uuid-file=/var/run/coreos/cfn-signal.uuid || :
 
+      # Filter out benign failures to signal stack
+      if [ $result_code -ne 0 ]; then
+        case $result in
+          *"for resource Controllers already exists"*)     exit 0
```

Comment:
I can't remember what was this message and when it happens 🤔 Would you mind providing me an example?

Reply:
It comes up if the message has already been sent before the cloudformation update is complete, so I saw it when I was testing the script for the failure responses.  We should probably squash both of them and have the service succeed should someone restart the cfn-signal service or the node for some reason.

Reply:
Ah, got it! Thanks for the clarification.